internal/driver/lanzou/share.go

```diff
@@ -41,9 +44,11 @@ func (l *Drive) removeNotes(html string) string {
 	return html
 }
 
-func (l *Drive) resolveFileShareURL(parsedURI string, pwd string) (*Response, error) {
-	get, _ := l.client.R().Get(parsedURI)
-
+func (l *Drive) resolveFileShareURL(parsedURI string, pwd string) (*ResponseData, error) {
+	get, err := l.client.R().Get(parsedURI)
```

Comment:
```suggestion
	resp, err := l.client.R().Get(parsedURI)
```

Use `resp` for same naming.